layers/+source-control/github/README.org

```diff
@@ -55,6 +58,10 @@ dotfile:
   (dotspacemacs-additional-packages '((forge :toggle t)))
 #+END_SRC
 
+** grip-mode
+Grip-mode [[https://github.com/seagle0128/grip-mode#prerequisite][requires python and the python package grip]] to be installed on the
+system. Grip can usually be installed with the command =pip3 install grip=.
```

Comment:
This should definitely refer to "pip" not "pip3". "pip3" is only available on debian based systems.

Reply:
Never mind I will change it myself